secret/secret_test.go

```diff
@@ -120,58 +119,52 @@ func (m *MockSecrets) RecordSSLCertInfo(configVar *model.ConfigurationVariable)
 	m.Called(configVar)
 }
 
-func (m *MockSecrets) GenerateSSLCerts(secrets, updates *v1.Secret) (dirty bool) {
-	results := m.Called(secrets, updates)
-	return results.Bool(0)
+func (m *MockSecrets) GenerateSSLCerts(secrets, updates *v1.Secret) {
+	m.Called(secrets, updates)
 }
 
 func TestUpdateSecretsWhenCreatingOrUpdating(t *testing.T) {
 	t.Parallel()
 
 	t.Run("Neither Create Nor Update called", func(t *testing.T) {
 		var s MockSecretInterface
```

Comment:
Should there be a `t.Parallel()` call here? (So that the various subtests can be run parallel to each other)